juju/framework.py

```diff
@@ -723,3 +763,19 @@ def __iter__(self):
 
     def __len__(self):
         return len(self._under)
+
+    def __le__(self, other):
+        if isinstance(other, set):
+            return self._under <= other
+        elif isinstance(other, StoredSet):
+            return self._under <= set(other)
```

Comment:
It seems like these should be swapped and the first one use `collections.abc.Set` instead of the built-in.

Reply:
Right, thanks for noticing.